util/perl/OpenSSL/Test.pm

```diff
@@ -66,7 +66,7 @@ use File::Spec::Functions qw/file_name_is_absolute curdir canonpath splitdir
                              rel2abs/;
 use File::Path 2.00 qw/rmtree mkpath/;
 use File::Basename;
-use Cwd qw/abs_path/;
+use Cwd qw/getcwd abs_path/;
```

Comment:
Just one question for curiousity: why do you sometimes use `abs_path()`, and sometimes `rel2abs()`?

Reply:
Regarding `abs_path`, I blame @mpeylo :wink:
We could absolutely use `rel2abs` instead